lib/packet.ml

```diff
@@ -845,11 +845,14 @@ type question = {
   q_unicast : q_unicast;
 }
 
+let make_question ?(q_class=Q_IN) ?(q_unicast=Q_Normal) q_type q_name =
```

Comment:
Comparing `make_`, `create_`, and `new_` across the codebase, I find that `make_` and `create_` are relatively unused but the `Loader` and `Trie` modules use `new_`. These modules' types are also mutable, however, so I'm not sure if this should be `new_`. I think `make_` is probably OK.